samples/mediator.ts

```diff
@@ -96,11 +96,11 @@ app.get('/api/connections', async (req, res) => {
   res.json(connections)
 })
 
-app.get('/api/routes', async (req, res) => {
+/*app.get('/api/routes', async (req, res) => {
   // TODO This endpoint is for testing purpose only. Return mediator connection by their verkey.
-  const routes = agent.routing.getRoutingTable()
+  const routes = agent.mediator.getRoutingTable()
   res.send(routes)
-})
+})*/
```

Comment:
Should we remove this?

Reply:
That is code from the previous implementation that will most likely not be needed but is commented out for quick references. I will convert dead code to comment anchors like TODO:'s.